CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Development
+
+## Development install
+
+First install `dev_dependencies`:
+
+```bash
+yarn
+```
+
+Build the library
+
+```bash
+yarn build
+```
+
+Run the test suite
+
+```bash
+yarn test
+```
+
+Start the example server at [`http://localhost:8080/examples/`](http://localhost:8080/examples/)
+
+```bash
+yarn start
+```
+
+## How to rebuild the Typescript declarations
+
+### Step-by-step
+
+1. For the duration of building declarations, temporarily `export` the
+`RegularTableElement` in `src/js/index.js`:
+
+```javascript
+// TEMP: export keyword added for building declarations
+export class RegularTableElement extends RegularViewEventModel {
```

Comment:
Okay, since this was the winner for "least automatable declarations build step", this change will also maybe make it easier for me to come up with a fully automated build for `index.d.ts`